packages/ttpassgen/PKGBUILD

```diff
@@ -0,0 +1,31 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=TTPassGen
+pkgver=133.a06d99d
+pkgrel=1
+pkgdesc='is a highly flexiable and scriptable password dictionary generator base on Python, you can easily use various rules to generate the desired combination of words.'
```

Comment:
pkgdesc='Highly flexiable and scriptable password dictionary generator base on Python, you can easily use various rules to generate the desired combination of words.'